manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,96 @@ products:
 
 # Rollover [index-rollover]
 
-When indexing time series data like logs or metrics, you can’t write to a single index indefinitely. To meet your indexing and search performance requirements and manage resource usage, you write to an index until some threshold is met and then create a new index and start writing to it instead. Using rolling indices enables you to:
+In {{es}}, the [rollover action](elasticsearch://reference/elasticsearch/index-lifecycle-actions/ilm-rollover.md) replaces your active write index with a new one whenever your index grows too large, too old, or stores too many documents.
```

Comment:
```suggestion
In {{es}}, the [rollover action](elasticsearch://reference/elasticsearch/index-lifecycle-actions/ilm-rollover.md) replaces your active write index with a new one whenever your index grows beyond a specified size, age, or number of documents.
```
Just a thought. I think someone might interpret "too large", "too old", etc. as meaning that Elasticsearch struggles to manage the index in these cases. Perhaps we can make the phrasing more positive, suggesting that ILM gives users control over these dimensions in order to optimize cluster performance and data retention to their needs.